cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/MetaDataApiModel.kt

```diff
@@ -71,11 +89,21 @@ data class MetaDataArg(
         @SerialName("hasLocalData") val hasLocalData: Boolean? = null,
         @SerialName("groupPmId") val groupPmId: String? = null,
         @SerialName("targetingParams") val targetingParams: JsonElement? = null,
-        @SerialName("uuid") val uuid: String? = null
+        @SerialName("uuid") val uuid: String? = null,
+    )
+
+    @Serializable
+    data class UsNatArg(
+        @SerialName("applies") val applies: Boolean?,
+        @SerialName("hasLocalData") val hasLocalData: Boolean? = null,
+        @SerialName("groupPmId") val groupPmId: String? = null,
+        @SerialName("targetingParams") val targetingParams: JsonElement? = null,
+        @SerialName("uuid") val uuid: String? = null,
```

Comment:
Where is `MetaDataArg` used? We should have `uuid` in it (unless this is the `metadata` query param of `consent-status` call.

Reply:
it is used in the message request